corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -240,9 +255,13 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                 answers = {},
                 model = this.parentModel;
             $fields.each(function (index) {
-                if (this.value !== '') {
-                    answers[model[index].get('id')] = encodeValue(model[index], this.value);
+                if (this.tagName === 'SELECT' && this.multiple) {
+                    var val = $(this).val();
+                }
+                else if (this.value !== '') {
+                    val = this.value;
```

Comment:
I don't see `val` declared outside of an if statement block. If neither of the if statements match won't `val` be undefined or cause an error?

Reply:
Updated